### DISCUSSION TITLE
Integrate WinMerge into Github Desktop

### DISCUSSION THREAD

**Discussion by SanSiets73** [community/community#75356] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear reades,

I want to integrate WinMerge into Github Desktop.
I've found old articles on the internet but they don't seem to work.
Can you please help me out?

**Comment by b8zeek** [comment, marked as answer]
Hey Sander. 👋🏼 

I am checking that right now, it seems straight forward... In order to integrate WinMerge into GitHub Desktop, you need to set WinMerge as the default diff tool. The steps seem to go:

Open GitHub Desktop ➡️ `File` ➡️ `Options` ➡️ Click the `Advanced tab` ➡️ In the `External Editor` section, select `Custom` ➡️ In the `Select Executable` dialog box, navigate to the location of your WinMerge executable and select it ➡️ `Save`.

Now it should work, when you select `Repository` and `Open in Command Prompt in GitHub Desktop` and type `git diff`, it should open WinMerge.

One more thing, I am reading that WinMerge is not officially supported by GitHub and that might be the reason for this issues, bare that in mind.

Good luck! 🍺  Let us know how this ended.

🍀
